Api/Entities/Story.cs

```diff
@@ -10,5 +10,6 @@ public class Story
         public string? Category { get; set; }
         public int AgeSuggested { get; set; }
         public string? Description { get; set; }
+        public string? ImageUrl { get; set; }    
```

Comment:
Remove null conditional operator from all models and entities that must have a value, the ones that can have a value optionally, could use the operator.